scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -204,6 +205,22 @@ workflow GvsCreateFilterSet {
         fq_filter_sites_destination_table = fq_filter_sites_destination_table,
         project_id = project_id
     }
+
+    call Utils.UberMonitor as UberMonitorLite {
```

Comment:
Maybe `UberMonitorVQSRLite` and `UberMonitorVQSRClassic` so it's not confusing what's Lite or Classic?

Reply:
No longer relevant here.